rules/windows/process_creation/proc_creation_win_deviceenroller_evasion.yml

```diff
@@ -0,0 +1,26 @@
+title: DLL Sideloading via DeviceEnroller.exe
+id: e173ad47-4388-4012-ae62-bd13f71c18a8
+description: Recent research discovered that DeviceEnroller.exe will load the DLL ShellChromeAPI.dll which does not exist
```

Comment:
It is more
 "
Detect use of the parameter PhoneDeepLink to SideLoading....
Recent research....
"